pkg/storage/bloom/v1/bloom_tokenizer.go

```diff
@@ -157,7 +156,7 @@ func (bt *BloomTokenizer) PopulateSeriesWithBloom(seriesWithBloom *SeriesWithBlo
 		)
 		if err != nil {
 			level.Info(util_log.Logger).Log("chunk iterator cannot be created")
-			return
+			return err
```

Comment:
Unrelated, but because I see it here: structured logging requires key-value pairs.

```
level.Info(util_log.Logger).Log("msg", "chunk iterator cannot be created")
```

Reply:
Indeed, fixed. thanks